modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc

```diff
@@ -146,3 +138,8 @@ spec:
 * Navigate to the `PipelineRun` details page and review the *Vulnerabilities* row for a visual representation of identified vulnerabilities.
 
 * Alternatively, you can navigate to the `PipelineRun` list view page, and review the *Vulnerabilities* column.
+
+[role="_additional-resources"]
+.Additional resources
+
+For more customized solution and information, see link:https://developers.redhat.com/products/trusted-application-pipeline/overview[Red Hat Trusted Application Pipeline].
```

Comment:
How about:
link:https://developers.redhat.com/products/trusted-application-pipeline/overview[Red Hat Trusted Application Pipeline (RHTAP) combines OpenShift pipelines with other Red Hat products to provide a secure, automated, customizable, end to end solution for building applications.]

Reply:
I feel an additional link to our docs here: https://docs.redhat.com/en/documentation/red_hat_trusted_application_pipeline/1.0/html-single/installing_red_hat_trusted_application_pipeline/index#idm139708457896560 could be useful to provide 
 more context

Reply:
Thank you so much for your suggestions.
I have updated the Additional resources section. Also, for the additional context, I added the Getting started guide link since it encompasses everything from Install to software templates.
Let me know if you have any additional suggestions or questions.